src/aws/mod.rs

```diff
@@ -43,10 +43,19 @@ pub fn is_retriable_error<T>(error: &SdkError<T>) -> bool {
     match error {
         SdkError::TimeoutError(_) | SdkError::DispatchFailure(_) => true,
         SdkError::ConstructionFailure(_) => false,
-        SdkError::ResponseError(err) => check_response(err.raw()),
-        SdkError::ServiceError(err) => check_response(err.raw()),
+        SdkError::ResponseError(err) => {
+            let raw = err.raw();
+
+            check_response(raw)
+        }
+        SdkError::ServiceError(err) => {
+            let raw = err.raw();
+
+            check_response(raw)
+        }
         _ => {
-            warn!("AWS returned an unhandled error, retrying request.");
+            // TODO make sure this message is useful for users
+            warn!("Unhandled SdkError please report this, retrying request");
```

Comment:
```suggestion
            warn!("AWS returned an unhandled error, retrying request.");
```

Unless you have a better suggestion on this log